packages/KTable/KTable.vue

```diff
@@ -110,8 +110,7 @@ table.k-table {
       }
       button,
       .k-button {
-        --KButtonPaddingY: 0;
-        --KButtonPaddingX: 0;
+        margin: calc(-1 * var(--KButtonPaddingY, var(--spacing-xs))) 0;
```

Comment:
not sure we want 0 margin on left and right

Reply:
yes you're right oversight. Just pushed to only do top and bottom